sopel/modules/currency.py

```diff
@@ -4,43 +4,43 @@
 # Licensed under the Eiffel Forum License 2
 from __future__ import unicode_literals, absolute_import, print_function, division
 
-import time
 import re
 import requests
+import time
 
-from sopel.module import commands, example, NOLIMIT, rule
-from sopel.tools import stderr
 from sopel.config.types import StaticSection, ValidatedAttribute
+from sopel.logger import get_logger
+from sopel.module import commands, example, NOLIMIT, rule
 
-rates_fiat_json = {}
-rates_btc_json = {}
-
-
-fiat_url = 'https://api.exchangeratesapi.io/latest?base=EUR'
-fixer_url = 'http://data.fixer.io/api/latest?base=EUR&access_key={}'
-crypto_url = 'https://apiv2.bitcoinaverage.com/indices/global/ticker/short?crypto=BTC'
-regex = re.compile(r'''
+FIAT_URL = 'https://api.exchangeratesapi.io/latest?base=EUR'
+FIXER_URL = 'http://data.fixer.io/api/latest?base=EUR&access_key={}'
+CRYPTO_URL = 'https://apiv2.bitcoinaverage.com/indices/global/ticker/short?crypto=BTC'
+EXCHANGE_REGEX = re.compile(r'''
     ^(\d+(?:\.\d+)?)                                            # Decimal number
     \s*([a-zA-Z]{3})                                            # 3-letter currency code
     \s+(?:in|as|of|to)\s+                                       # preposition
     (([a-zA-Z]{3}$)|([a-zA-Z]{3})\s)+$                          # one or more 3-letter currency code
 ''', re.VERBOSE)
 
```

Comment:
This blank line is for…? If anything, put a blank between `LOGGER` (a "constant") and the `rates_*_json` cache dicts.